test/resources/channels.test.js

```diff
@@ -0,0 +1,15 @@
+
+const factory = require('../../lib/resources/channels')
+
+const BASE = '/channels'
+describe(BASE, () => {
+  const req = {
+    get: jest.fn()
+  }
+  const client = factory(req)
+
+  it(`GET ${BASE}`, () => {
+    client.get()
```

Comment:
Thanks for your PR! I would just add async/await here since the `.get()` returns a promise. Otherwise looks great.

Reply:
Done! :)